doc/index.rst

```diff
@@ -82,15 +34,6 @@ Announcements
 *************
 
 - :ref:`DIPY 0.14 <release0.14>` released May 1, 2018.
-- :ref:`DIPY 0.13 <release0.13>` released October 24, 2017.
-- :ref:`DIPY 0.12 <release0.12>` released June 26, 2017.
```

Comment:
At least, let's keep the last 3 announcements/events. Can you put `release 0.13` and `0.12` back?